src/sinol_make/oiejq/__init__.py

```diff
@@ -14,13 +14,21 @@ def _check_if_oiejq_executable(path):
         return False
 
     try:
-        p = subprocess.Popen([path], shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-        p.wait()
-        return p.returncode == 0
+        oiejq = subprocess.Popen([path], shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        oiejq.wait()
+        return oiejq.returncode == 0
     except FileNotFoundError:
```

Comment:
Is this really needed?
When testing outside of sinol-make, I couldn't trigger an exception.

Reply:
That's true, in theory here an exception shouldn't happen. I'll delete it